bindings/nodejs/tests/test_service.test.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { randomUUID } from 'node:crypto'
+import { expect, test } from 'vitest'
+import { Operator } from '../index.js'
+import { generateBytes } from './utils.js'
+
+const operator = new Operator(process.env.SCHEME, { root: '/tmp' })
```

Comment:
I prefer using `OPENDAL_TEST` over `SCHEME`. Another issue is that the configuration isn't loading from the environment. We need to load all environments starting with `OPENDAL_XXX_{KEY}` as configuration here.